### ISSUE TITLE
Variable length fields

### ISSUE THREAD

**Issue by FreakyPenguin** [p4lang/p4c-behavioral#1]
As far as I can tell, the behavioral model seems to miss support for variable length fields. From a quick look at the source code it looks like adding this would mean moving some stuff around. Is anyone working on adding this, or is this a lower priority thing?

In general it would be useful to outline what's missing somewhere, e.g. in the README.

Thanks

**Comment by antoninbas** [comment]
You are right, there is no current support for variable length headers in the behavioral model and I don't have time to add support in the near future. Of course anyone is welcome to contribute a patch, but I think supporting this feature would require some big changes.
Will try to update the README with current limitations of the model soon.
